src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -100,8 +100,7 @@ public class CookieContainer
             new HeaderVariantInfo(HttpKnownHeaderNames.SetCookie2, CookieVariant.Rfc2965)
         };
 
-        // NOTE: all accesses of _domainTable must be performed with _domainTable locked.
-        private readonly Dictionary<string, PathList> m_domainTable = new Dictionary<string, PathList>(); // Do not rename (binary serialization)
+        private readonly Hashtable m_domainTable = new Hashtable(); // Do not rename (binary serialization)
```

Comment:
Because Desktop uses Hashtable and we are using vanilla binary serialization so the types and names must match.